test/src/org/exist/xupdate/modifications/append_child.xml

```diff
@@ -5,7 +5,7 @@
 		<xu:element name="email"><xu:attribute name="type">home</xu:attribute>andrew@foo.org</xu:element>
 	</xu:append>
 	
-	<xu:append select="//address[lname = 'Jekyll']" child="3">
+	<xu:append select="//address[lname = 'Jekyll']" child="5">
```

Comment:
I will ofcourse look into if there are other issues with XUpdate in the debugger, but for these tests all whitespace is explicitly stripped in the test code so I suspect little taint from my change.